### ISSUE TITLE
Context-sensitive requirements

### ISSUE THREAD

**Issue by MartinKasp** [IHE/sdpi-fhir#278]
## Section Number

Entire document

## Priority

Low to high ;-)

## Issue

Some requirements are context-sensitive to the chapter/annex the belong to. Thus, when you only get rie requirement, it can be misleading. For example: R1002 forbids UPD connections (except for ad-hoc discovery). However, this is only valid for the MDPWS binding. Thus, you have to know that R1002 is part of appendix A to understand it. The requirement will not apply for, e.g., BICPES-over-CoAP.

## Proposed Change

Intended fix: In general, we should aim for requirements that can be interpreted unambiguously without context or that the context is always clear. Options: Rephrase requirements; introduce requirement number prefixes that clarify the context unambiguously.

**Comment by ToddCooper** [comment]
Indeed this is a broader, longer term discussion than what we need to accomplish in SDPi 1.0 for Public Comment and even Trial Implementation.  Thus, I added the label "SDPi 1.x" to make sure it gets discussed but that a complete reworking of requirements language does not have to be completed before moving forward.

Language in TF-1 Appendix A section _**A.6 SDPi Requirements Modeling & Integration**_ indicates that for SDPi 1.0, RI is an initial step and that it will be expanded in subsequent versions (at least that's what it WILL say when that section is finalized for Public Comment).  

Requirement scoping & context should be easily achievable with sufficient guidelines & spec data / metadata, including the Proposed Change.  Note: this challenge is why many specifications integrate (at least in metadata) the section in which the requirement is formalized.  Ugly to be sure, but a necessary evil in most cases.

NOTE:  Review the SysML 2.0 approach to requirements specification language.

**Comment by AnnaFeiler** [comment]
@MartinKasp: This requires some more discussion. Maybe we can group requirements by the corresponding actors, provide ICS tables,... Since we don't have mutually exclusive bindings yet, can we postpone this to 1.x?

**Comment by ToddCooper** [comment]
This discussion should be rooted in the content in TF-1 Appendix A "requirements modeling" sections.

**Comment by ToddCooper** [comment]
Confirmed as deferred to 1.x on Friday call 03.02.2023

**Comment by MartinKasp** [comment]
> @MartinKasp: This requires some more discussion. Maybe we can group requirements by the corresponding actors, provide ICS tables,... Since we don't have mutually exclusive bindings yet, can we postpone this to 1.x?

agreed

**Comment by AnnaFeiler** [comment]
deferred but not completed
